### DISCUSSION TITLE
Limit number of eslint-disable lines?

### DISCUSSION THREAD

**Discussion by jsjoeio** [eslint/eslint#16461] [Q&A] [answered]
Is there a way to limit the number of `//eslint-disable` lines in a codebase?

I've been fixing some here and there in my team's codebase but I want to prevent more from being added.

**Comment by jsjoeio** [comment, marked as answer]
Someone on Twitter suggested this: https://mysticatea.github.io/eslint-plugin-eslint-comments

**Reply by nzakas** [reply]
Yup, that's the best way to do it. There's no built-in facility for doing this with ESLint.
